### PR TITLE
add route to display assessment navigation sidebar

### PR DESCRIPTION
**Pull request by Adam-W1**
https://digital.dclg.gov.uk/jira/browse/FS-1929

The navigation sidebar when viewing an assessment's sub criteria.

Linked to: https://github.com/communitiesuk/funding-service-design-assessment-store/pull/54

TODO: The navigation sidebar will sit within a larger template as a macro.

### CODE REVIEW

File: config/envs/default.py [hunk 1]
```diff
@@ -52,11 +52,10 @@ class DefaultConfig:
     APPLICATION_SEARCH_ENDPOINT = CommonConfig.APPLICATION_SEARCH_ENDPOINT
 
     # Assesment store endpoints
-    ASSESSMENT_STORE_API_HOST = CommonConfig.ASSESSMENT_STORE_API_HOST
```
Comment (Adam-W1) [inline config/envs/default.py:55]: this was duplicated

File: tests/test_routes.py [hunk 2]
```diff
@@ -141,3 +141,19 @@ def test_route_landing_clear_filters(self, flask_test_client):
             assert (
                 b"Assessor dashboard</p>" in response.data
             ), "Response does not contain expected heading"
+
+    @pytest.mark.parametrize(
+        "expected_strings",
+        [b"general-information", b"activities", b"partnerships"],
```
Comment (gidsg) [inline tests/test_routes.py:147]: Should we check for the names as well as the IDs?

File: app/assess/forms/comments_form.py [hunk 3]
```diff
@@ -4,6 +4,5 @@
 
 
 class CommentsForm(FlaskForm):
-    
-        comment = TextAreaField("Comment",validators = [length(max=200)])
-        
\ No newline at end of file
+
+    comment = TextAreaField("Comment", validators=[length(max=200)])
```
Comment (gidsg) [inline app/assess/forms/comments_form.py:8]: 200 characters seems very small for a comment. Can we check with the designers, but I think they didn't want a limit possibly, so maybe set an arbitrarily large limit like 10000?

File: tests/test_routes.py (hunk 2 above)
Comment (Adam-W1) [reply tests/test_routes.py]: good shout, added

File: app/assess/forms/comments_form.py (hunk 3 above)
Comment (Adam-W1) [reply app/assess/forms/comments_form.py:8]: This was a python reformatted file unrelated to the PR so unsure on why it's been set to 200, ill bump up the limit 👍

File: app/assess/models/sub_criteria.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import inspect
+from dataclasses import dataclass
+from app.assess.models.theme import Theme
+
+
```
Comment (tferns) [inline app/assess/models/sub_criteria.py:5]: Is it worth using the helpful `dataclass_dict_convert` here as well?

**Review by tferns — APPROVED** [review]
Looks good! 👍

**Pull request merged**